CHANGES.txt

```diff
@@ -4,8 +4,11 @@ Changes
 2.6 (unreleased)
 ----------------
 
-- Nothing changed yet.
-
+- compatibility for python 3
+- update code to work with python2 and python3
+- adjust egg to work with newer version of setuptools
+- reimplement ``zope.interface.implements`` with a grokker
+  for both python2 and python3
```

Comment:
Shouldn't this be "reimplement ``grok.implements`` with a grokker"?

(I'll postpone the release until tomorrow so the changelog can get a chance to be clarified.)